tools/encode_program_info/encode_program_info.cpp

```diff
@@ -6,6 +6,12 @@
 #include "ebpf_nethooks.h"
 #include "encode_program_info.h"
 
+extern "C"
```

Comment:
This file needs to be reverted.    PR#492 put the correct defines into encode_program_info.h which is included at line 7.
Below has the incorrect defines with the _ prefix.

Reply:
fixed.